src/providers/lemonfox/index.ts

```diff
@@ -0,0 +1,33 @@
+import { ProviderConfigs } from '../types';
+import LemonfoxAIAPIConfig from './api';
+import {
+  LemonfoxAIChatCompleteConfig,
+  LemonfoxAIChatCompleteResponseTransform,
+  LemonfoxAIChatCompleteStreamChunkTransform
+} from './chatComplete'
+import {
+  LemonfoxAIImageGenerateConfig,
+  LemonfoxImageGenerateResponseTransform,
+
+} from './imageGenerate'
+
+import {
+  LemonfoxAICreateTranscriptionResponseTransform,
+  LemonfoxAIcreateTranscriptionConfig
+} from './createTranscription'
+
+const LemonfoxAIConfig: ProviderConfigs = {
+  chatComplete: LemonfoxAIChatCompleteConfig,
```

Comment:
Since chat completions for this provider is open-ai compliant, you can use the transformers from `openai-base` folder. Checkout `inference-net` and `sambanova` for reference implementations

Reply:
done